.github/scripts/end2end/run-e2e-test.sh

```diff
@@ -120,7 +120,35 @@ run_e2e_test() {
         --env=MONGO_WRITE_CONCERN=${MONGO_WRITE_CONCERN} \
         --env=MONGO_AUTH_USERNAME=${MONGO_AUTH_USERNAME} \
         --env=MONGO_AUTH_PASSWORD=${MONGO_AUTH_PASSWORD} \
-        --command -- sh -c "${2}"
+        --override-type strategic \
+        --overrides='
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "spec": {
+    "containers": [
+      {
+        "name": "'$POD_NAME'",
+        "volumeMounts": [
+          {
+            "name": "artifacts",
+            "mountPath": "/artifacts"
+          }
+        ]
+      }
+    ],
+    "volumes": [
+      {
+        "name": "artifacts",
+        "hostPath": {
+          "path": "/data/artifacts",
```

Comment:
`/data` works in CI, but most likely not if the test were to run locally: should instead rely on $(pwd) or something similar (which may possibly be overridden...)

```suggestion
          "path": "${ZENKO_E2E_ARTIFACTS:-$(pwd)}/artifacts",
```

Reply:
Ok but I cannot use this var, as we need to be consistent with [the one defined inside `bootstrap-kind.sh`](https://github.com/scality/Zenko/blob/feature/ZENKO-4804-upload-test-results-trunk/.github/scripts/end2end/bootstrap-kind.sh#L6). I also can't use `$pwd` as a path prefix.

Reply:
Actually I had to revert that change because `/data/artifacts` is a reference to a already hardcoded variable, we are referring the host path, in this case it is the container kind, and when bootstrapping kind we refer the host path there as well, which is the machine that will execute the tests, CI or local. So in the container context, `/data/` is bind to where you will specify `VOLUME_ROOT` when bootstrapping kind, which is wherever the user decides to do so.

Too many layers 😅

Reply:
🤯